src/displayList.js

```diff
@@ -0,0 +1,94 @@
+import { getLocalStorage, setLocalStorage } from './LocalStorage';
```

Comment:
[REQUIRED] I can't find a function for editing the description of your tasks. Kindly include it.